socialmediapolicy.md

```diff
@@ -0,0 +1,155 @@
+* All posts must be considered public, regardless of where they are posted, and be treated as such.
+* Employees must only upload media that they themselves have created and own the IP (intellectual property) to.
+* The school must keep a list of children who do not have parental consent to be online. This must be cross referenced before every post.
+*	Employees must not post anything which may bring the school into disrepute or portray the school in a negative light.
+
+#### Should (recommended)
+* There should not be any content posted by the school to a private page or closed group.
+* Employees should not post about any school related matters on their own personal account.
+
+#### May (optional)
+* Employees may like, share, favourite, comment (or similar) on school related content that has been published via official school accounts.
+* Employees may want to consider the time of day they are posting to maximise their audience.
+
+### Engaging with Parents and Pupils
+
+#### Must (required)
+* Employees must ensure they do not interact or post comments on Social Networks which may bring themselves or the school into disrepute.
+
+#### Should (recommended)
+* Employees should not connect or engage with any pupils on the current school roll via Social Media.
+* Employees should only engage with parents, pupils and the public with regard to school matters via official school accounts.
+
+#### May (optional)
+* Employees may connect with parents/guardians of a pupil currently on roll if they are known on a personal level.
+* Employees may engage and reply where possible via the official school accounts to increase engagement with others.
+*	Employees may share the content posted to official school accounts on their own accounts if they wish to.
+
+### Passwords and Security
+
+##### Must (required)
+*	Passwords to school social networking accounts must remain safe and secure at all times.
+*	Passwords must not be shared between employees.
+
+##### Should (recommended)
+*	Social Networks should be managed either through a single person or via a social media management solution in which all users will have their own log in and password.
+* Employees should ensure that content is checked by another member of the team before posting to ensure the content abides by the rules contained within this policy.
+
+#### May (optional)
+* Employees may want to set up two factor authentication on their social media accounts for added security.
+
+### Intellectual Property (IP)
+
+##### Must (required)
+*	All media posted must be owned by the person uploading, or the organisation they work for, before it can be uploaded.
+*	Unless stated otherwise in an employment contract, all media and IP (intellectual property) remains the property of the person who created the media. By posting the content to a school account, permission is granted to the school to use the media under the Creative Commons License.
```

Comment:
We should remove the CC bit for now, I am not sure the licence fits our needs after reading it in detail as you have to provided Attribution.

I would adjust that by posting they give school permission to share (copy and redistribute the material in any medium or format) and adapt (remix, transform, and build upon the material) the artwork.

This may need some more thought.